tests/core/mountpoint/test_fuse.py

```diff
@@ -189,3 +190,14 @@ async def _mocked_bootstrap_mountpoint(*args):
 
         # Test is over, stop alice2 mountpoint and exit
         nursery.cancel_scope.cancel()
+
+
+@pytest.mark.linux
```

Comment:
My guess is that it's more complicated than that since windows sometimes deal with long path, as far as I remember this limitation is on the application side depending on which api it's using in order to access the file system.

I can run a quick test.

Reply:
I can confirm that accesses to files with names over 255 bytes do not reach the `winfsp_operations` module. Weirdly enough, the version with ascii characters return with an "Invalid argument" error while the version with unicode characters return with a "File not found" error. Hence I'm keeping this test in the fuse module.

Reply:
475c2bc54ef61116cd43962f145610d6d5bbe567